### PR TITLE
increase map complexity limits

### PR DESCRIPTION
**Pull request by inf1niti**
Some arbitrary increases to hardcoded limits in attempt to fix some of the issues I've run into on larger maps.

Not all of these fixed the problems unfortunately :/ still was able to compile and run this without any problems.  Main thing that didn't work for me was I was still hitting max efrags & max channels (sound) limits in rctf5 :(

### CODE REVIEW

File: r_texture_load.c [hunk 1]
```diff
@@ -556,7 +556,7 @@ static void R_Upload32(gltexture_t* glt, unsigned *data, int width, int height,
 
 static void R_Upload8(gltexture_t* glt, byte *data, int width, int height, int mode)
 {
-	static unsigned trans[640 * 480];
+	static unsigned trans[1024 * 1024];
```
Comment (inf1niti) [inline r_texture_load.c:559]: this one works and caused no issues for me locally 👍 
If you're unsure about any of the other changes, you can reduce this to a 1 liner and should be able to support all the common textures found in popular wads today.

File: common.h [hunk 2]
```diff
@@ -70,11 +70,11 @@ typedef enum
 // per-level limits
 #define CL_MAX_EDICTS           2048	// FIXME: ouch! ouch! ouch!
 #define MAX_EDICTS              2048    // can't encode more than this, see SV_WriteDelta
-#define MAX_EDICTS_SAFE         512     // lower limit, to make sure no client limits exceeded
+#define MAX_EDICTS_SAFE         1024     // lower limit, to make sure no client limits exceeded
 #define MAX_LIGHTSTYLES         64
 #define MAX_MODELS              512	    // can't encode more than this, see SV_WriteDelta
 #define MAX_VWEP_MODELS         32	    // could be increased to 256
-#define MAX_SOUNDS              256	    // so they cannot be blindly increased
+#define MAX_SOUNDS              512	    // so they cannot be blindly increased
```
Comment (inf1niti) [inline common.h:77]: didn't fix the rctf5 issue, unfortunately :(

File: client.h [hunk 3]
```diff
@@ -303,8 +303,8 @@ typedef struct {
 	char	map[MAX_STYLESTRING];
 } lightstyle_t;
 
-#define	MAX_EFRAGS			512
-#define	MAX_STATIC_ENTITIES	512		// torches, etc
+#define	MAX_EFRAGS			1024
```
Comment (inf1niti) [inline client.h:306]: still get too many efrags on rctf5 with this change

**Comment by dsvensson** [comment]
merged

**Pull request closed without merging**